src/templates/static/legal/tac_sections/tac.jsx

```diff
@@ -2,134 +2,139 @@ import React from 'react';
-        <p data-show='eucountry'>{it.L('The company reserves the right to charge a dormant fee of up to USD/EUR/GBP 25 every 6 months for every Client account that has had no transaction recorded for over 12 months. Clients will be notified before the fee is deducted from the account. Clients need to contact our helpdesk to recover funds from inactive, closed, blocked or excluded accounts.')}</p>
+        <h2 data-anchor='dormant-and-inactive-accounts'>K. {it.L('Dormant and inactive accounts')}</h2>
+        <p data-show='-eucountry'>{it.L('The company reserves the right to charge a dormant fee of up to USD/EUR/AUD/GBP 25 or, for the cryptocurrency accounts, the equivalent of USD 25 as calculated in accordance with the current exchange rate published by www.xe.com, or another recognised data provider, as on the date of charging of the fee, every six months for every client\'s account that has had no transaction recorded for over 12 months.')}</p>
+        <p data-show='eucountry'>{it.L('The company reserves the right to charge a dormant fee of up to USD/EUR/GBP 25 every six months for every client\'s account that has had no transaction recorded for over 12 months. Clients will be notified before the fee is deducted from the account. Clients need to contact our helpdesk to recover funds from inactive, closed, blocked, or excluded accounts.')}</p>
         <h2 data-anchor='liability'>L. {it.L('Liability')}</h2>
-        <p>{it.L('Save in case of negligence, fraud, or default by the Company, the Company will not be liable in any way to any clients in the event of force majeure, or for the act of any Government or legal authority, or for the failure of or damage or destruction to its computer systems, data, or records or any part thereof, or for delays, losses, errors, or omissions resulting from the failure or mismanagement of any telecommunications, computer equipment or software by the client or any damages or losses deemed or alleged to have resulted from or been caused by this site or its content.')}</p>
+        <p>{it.L('Save in case of negligence, fraud, or default by the Company, the Company will not be liable in any way to any clients in the event of force majeure, or for the act of any government or legal authority, or for the failure of or damage or destruction to its computer systems, data, or records or any part thereof, or for delays, losses, errors, or omissions resulting from the failure or mismanagement of any telecommunications, computer equipment, or software by the client or any damages or losses deemed or alleged to have resulted from, or been caused by, this site or its content.')}</p>
```

Comment:
```suggestion
        <p>{it.L('Save in case of negligence, fraud, or default by the Company, the Company will not be liable in any way to any clients in the event of force majeure, or for the act of any government or legal authority, or for the failure of or damage or destruction to its computer systems, data, or records, or any part thereof, or for delays, losses, errors, or omissions resulting from the failure or mismanagement of any telecommunications, computer equipment, or software by the client or any damages or losses deemed or alleged to have resulted from, or been caused by, this site or its content.')}</p>
```